qui/tray/devices.py

```diff
@@ -78,14 +78,13 @@ def attach_item(self, menu_item):
                 "Attaching {} to {}".format(self.device.description,
                                             menu_item.vm),
                 Gio.NotificationPriority.NORMAL)
-        except qubesadmin.exc.QubesException as ex:
+        except Exception as ex:  # pylint: disable=broad-except
```

Comment:
For non-QubesException it may worth adding exception name too. While QubesException is used with human readable messages, others may not - for example `KeyError` often contain just the key.

Reply:
fixed in the fixup below